.github/workflows/run_test_windows.yml

```diff
@@ -0,0 +1,287 @@
+                  AZLE_IS_FEATURE_BRANCH_DRAFT_PR="${{ steps.set-conditions.outputs.is_feature_branch_draft_pr || 'false' }}"
+
+                  RECORD_ACTIONS='true'
+                  RUNS=1
+
+                  if [[ "$AZLE_IS_MAIN_BRANCH_PUSH_FROM_FEATURE_MERGE" == "true" ]]; then
+                      RUNS=5
+                  fi
+
+                  if [[ "$AZLE_IS_MAIN_BRANCH_PUSH_FROM_RELEASE_MERGE" == "true" ]]; then
+                      RUNS=100
+                  fi
+
+                  if [[ "$AZLE_IS_RELEASE_BRANCH_PR" == "true" ]]; then
+                      RECORD_ACTIONS='false'
+                      RUNS=10
+                  fi
+
+                  if [[ "$AZLE_IS_FEATURE_BRANCH_PR" == "true" ]]; then
+                      RUNS=5
+                  fi
+
+                  if [[ "$AZLE_IS_FEATURE_BRANCH_DRAFT_PR" == "true" ]]; then
+                      RUNS=1
+                  fi
+
+                  echo "Running tests $RUNS times"
+
+                  echo "runs=$RUNS" >> $GITHUB_OUTPUT
+                  echo "record_actions=$RECORD_ACTIONS" >> $GITHUB_OUTPUT
+
+            - name: Run tests
+              run: |
+                  # Set clean PATH
+                  export PATH="/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:$HOME/.local/share/dfx/bin:$HOME/.cargo/bin"
+
+                  # Set workflow context variables inline
+                  export AZLE_IS_MAIN_BRANCH_PUSH_FROM_FEATURE_MERGE="${{ steps.set-conditions.outputs.is_main_branch_push_from_feature_merge || 'false' }}"
+                  export AZLE_IS_MAIN_BRANCH_PUSH_FROM_RELEASE_MERGE="${{ steps.set-conditions.outputs.is_main_branch_push_from_release_merge || 'false' }}"
+                  export AZLE_IS_RELEASE_BRANCH_PR="${{ steps.set-conditions.outputs.is_release_branch_pr || 'false' }}"
+                  export AZLE_IS_FEATURE_BRANCH_PR="${{ steps.set-conditions.outputs.is_feature_branch_pr || 'false' }}"
+                  export AZLE_IS_FEATURE_BRANCH_DRAFT_PR="${{ steps.set-conditions.outputs.is_feature_branch_draft_pr || 'false' }}"
+
+                  # Set test configuration variables
+                  export AZLE_EXPERIMENTAL="${{ inputs.run-experimental }}"
+                  export AZLE_FUZZ="${{ inputs.fuzz }}"
+                  export AZLE_FUZZ_CALL_DELAY="${{ inputs.call-delay }}"
+                  export AZLE_FUZZ_TIME_LIMIT="${{ inputs.time-limit }}"
+                  export AZLE_RECORD_ACTIONS="${{ steps.test-configuration.outputs.record_actions }}"
+                  export AZLE_PROPTEST_NUM_RUNS="${{ steps.test-configuration.outputs.runs }}"
+                  export AZLE_PROPTEST_VERBOSE=true
+                  export AZLE_VERBOSE=true
+                  export AZLE_IDENTITY_STORAGE_MODE=plaintext
+                  export GITHUB_ACTIONS=true
+                  export AZLE_TEST_WSL=true
+
+                  dfx start --clean --background --artificial-delay 0
```

Comment:
Can't we use the same steps and ordering as in the linx/mac run yaml? Can we keep the differences between them more minimal?